src/hera/workflows/io.py

```diff
@@ -0,0 +1,86 @@
+"""Input/output models for the Hera runner. TODO move to hera.workflows.runner package."""
```

Comment:
@samj1912 / @flaviuvadan any preference on where these classes should live? Considering they're only usable in the runner, thought a rearrangement of that module would work and move these classes into it?

Reply:
workflows/io is fine